jujugui/static/gui/src/app/models/handlers.js

```diff
@@ -110,6 +110,76 @@ YUI.add('juju-delta-handlers', function(Y) {
         result.tags = tags.join(',');
       }
       return result;
+    },
+
+    /**
+      Translates the new JujuStatus and WorkloadStatus's to the legacy Juju 1
+      values.
+
+      This logic is a JS implementation of the Juju core implementation to
+      accomplish the same task:
+      https://github.com/
+        juju/juju/blob/juju-1.26-alpha1/state/status_model.go#L272
+
+      @method translateToLegacyAgentState
+      @param {String} agentStatus JujuStatus.Current
+      @param {String} workloadStatus WorkloadStatus.Current
+      @param {String} workloadStatusMessage WorkloadStatus.Message
+    */
+    translateToLegacyAgentState: function(
+      agentStatus, workloadStatus, workloadStatusMessage) {
+      var statusMaintenance = 'maintenance';
```

Comment:
At first I thought so as well but I couldn't find anywhere that needed them without making the rest of the file cluttered. I imagine that we're going to have to rewrite this at some point anyways and at that point this will go somewhere better.